cdp-agentkit-core/src/actions/cdp/defi/wow/actions/create_token.ts

```diff
@@ -4,7 +4,18 @@ import { WOW_FACTORY_ABI, GENERIC_TOKEN_METADATA_URI, getFactoryAddress } from "
 import { z } from "zod";
 
 const WOW_CREATE_TOKEN_PROMPT = `
-This tool will create a Zora Wow ERC20 memecoin using the WoW factory. This tool takes the token name, token symbol, and optionally a token URI containing metadata about the token. It uses a bonding curve so there is no need to add liquidity to the pool upfront. It is only supported on Base Sepolia and Base Mainnet.
+This tool can only be used to create a Zora Wow ERC20 memecoin using the WoW factory. Do not use this tool for any other purpose, or creating other types of tokens.
```

Comment:
can we add bonding curve to the prompt. Many people refer to wow tokens as bonding curve tokens since that is the common underlying implementation.

Reply:
done. this will need to be added to AgentKit.py as well